<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFormWidget</class>
 <widget class="QWidget" name="UserFormWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="firstNameLe">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastNameLe">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ageLe">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="typeCb">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>VIP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Employee</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="designationCb">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ASE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SSE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DL</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="saveBut">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetBut">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBut">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>FirstName:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LastName:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Designation:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>resetBut</sender>
   <signal>clicked()</signal>
   <receiver>ageLe</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetBut</sender>
   <signal>clicked()</signal>
   <receiver>lastNameLe</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetBut</sender>
   <signal>clicked()</signal>
   <receiver>firstNameLe</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
